mycore-neo4j/src/main/java/org/mycore/datamodel/metadata/neo4jparser/MCRNeo4JAbstractDataModelParser.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * This file is part of ***  M y C o R e  ***
+ * See http://www.mycore.de/ for details.
+ *
+ * MyCoRe is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * MyCoRe is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with MyCoRe.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.mycore.datamodel.metadata.neo4jparser;
```

Comment:
Do not use `org.mycore.datamodel` as package root, because this is already part of `mycore-base`
We must start to keep things separate for the planed switch to Java modules
Use `org.mycore.neo4j` as root

Reply:
We selected the sub package because it is a directly connection between the parser classes and the datamodel classes. The neo4j datamodel classes use the structure of the MCRMetadataXYZ classes to extract neo4j data. So it seems the best solution to use this name.

Reply:
With the planed use of Java modules every JAR file needs a unique package structure.
We've decided, that new modules should already follow the rules (see *mycore-cronjob* or *mycore-acl*).
In your case it could be `org.mycore.mcr.neo4j.datamodel.metadata.neo4jparser`